x-pack/plugins/aiops/public/ui_actions/create_change_point_chart.tsx

```diff
@@ -39,6 +39,7 @@ export function createAddChangePointChartAction(
         getIconType: () => 'machineLearningApp',
       },
     ],
+    getIconType: () => 'machineLearningApp',
```

Comment:
@andreadelrio @joana-cps any chance we could come up with a more tailored icon for the change point detection panel type rather than using the generic ML icon? 

<img width="229" alt="Screenshot 2024-06-05 at 16 50 01" src="https://github.com/elastic/kibana/assets/7405507/54d7ae4b-9167-496a-8ff0-42a7f231737d">

Something similar to the `visLine` icon used for the anomaly charts, but perhaps showing a spike or a step change?

<img width="385" alt="Screenshot 2024-06-05 at 16 52 46" src="https://github.com/elastic/kibana/assets/7405507/5150e9c6-77fd-4649-935b-2fcc1a8c6d87">

Reply:
Sure, it's a really good point. I'll draft it up and share it with the team.